contracts/upgradeable_contracts/erc20_to_native/ForeignBridgeErcToNative.sol

```diff
@@ -63,4 +78,33 @@ contract ForeignBridgeErcToNative is BasicForeignBridge, ERC20Bridge {
     function onFailedMessage(address, uint256, bytes32) internal {
         revert();
     }
+
+    function _setBridgeContractOnOtherSide(address _bridgeContract) internal {
+        addressStorage[BRIDGE_CONTRACT] = _bridgeContract;
+    }
+
+    function bridgeContractOnOtherSide() public view returns (address) {
+        return addressStorage[BRIDGE_CONTRACT];
+    }
+
+    function _relayRequest(address _sender, address _receiver, uint256 _amount) internal {
+        require(_receiver != address(0));
+        require(_receiver != address(this));
+        require(_receiver != bridgeContractOnOtherSide());
+        require(_amount > 0);
+        require(withinLimit(_amount));
+        setTotalSpentPerDay(getCurrentDay(), totalSpentPerDay(getCurrentDay()).add(_amount));
+
+        erc20token().transferFrom(_sender, address(this), _amount);
+        emit UserRequestForAffirmation(_receiver, _amount);
+    }
+
+    function relayRequest(address _from, address _receiver, uint256 _amount) external {
```

Comment:
Do you think it makes sense to be consistent with the contracts implementing `erc677-to-erc677` on top of AMB and name the method `relayTokens`?

Reply:
Yes, I think we should be consistent. We are also using `relayRequest` for alternative receiver in the native side.  What do you think would be best, keeping `relayRequest` for native side and name `relayTokens` on erc20 sides or having the same name `relayRequest` for both native and erc20 sides? In the second case the name in  `erc677-to-erc677` on top of AMB should be updated

Reply:
Thanks. Renamed `relayRequest` to `relayTokens` https://github.com/poanetwork/poa-bridge-contracts/pull/294/commits/4fb9b9eb6c73fc71db489620879c79cf6865c644

I'll apply similar changes to the other PRs so `relayTokens` is used

Reply:
thanks